plugins/qtosgrave/qtosgviewer.cpp

```diff
@@ -1814,6 +1829,35 @@ GraphHandlePtr QtOSGViewer::drawtrimesh(const float* ppoints, int stride, const
     return GraphHandlePtr(new PrivateGraphHandle(shared_viewer(), handle));
 }
 
+class GlobalLOD : public osg::LOD
+{
+public:
+    void traverse(osg::NodeVisitor& nv)
+    {
+        osg::CullStack *cs;
```

Comment:
declare variable right before use (before line 1840) and always initialize it (nullptr in this case).